variants/heltec_wireless_paper_v1/variant.h

```diff
@@ -16,9 +16,9 @@
 
 // SPI
 #define SPI_INTERFACES_COUNT 2
-#define PIN_SPI_MISO 10 // MISO
-#define PIN_SPI_MOSI 11 // MOSI
-#define PIN_SPI_SCK 9   // SCK
+#define PIN_SPI_MISO 11
```

Comment:
I thought the wireless paper variants were still working?

Reply:
Oh that's just tidying up those defines. Apparently they're not used, but they didn't actually match the [real values just below](https://github.com/meshtastic/firmware/blob/bcdda4de8ab81fec6a35be52cc2a3a0fa3fa5332/variants/heltec_wireless_paper/variant.h#L40-L43)
```cpp
#define LORA_SCK 9
#define LORA_MISO 11
#define LORA_MOSI 10
#define LORA_CS 8
```

Reply:
Oh! I was confused for a second 😅